Lombiq.HelpfulLibraries/Libraries/Utilities/ArrayExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Linq;
+
+namespace System
+{
+    public static class ArrayExtensions
+    {
+        /// <summary>
+        /// Converts an <see cref="object"/> array to a <see cref="string"/> array.
+        /// </summary>
+        /// <param name="includeNulls">If <see langword="false"/>, the converted array will exclude nulls (i.e., the
+        /// that are not <see cref="string"/>s).</param>

```

Comment:
> the that are not strings

![image](https://user-images.githubusercontent.com/4007293/123962206-7d539300-d9b1-11eb-8988-591c3aad43ee.png)

Reply:
😄

Reply:
![image](https://user-images.githubusercontent.com/5919557/123972409-1c30bd00-d9bb-11eb-9b85-d3361d5aa360.png)

Reply:
Just noticed that I don't use this so I'll delete it. Easiest fix. :D